ctapipe_io_lst/tests/test_lsteventsource.py

```diff
@@ -243,3 +244,24 @@ def test_len():
         max_events=10,
     ) as source:
         assert len(source) == 10
+
+
+def test_pedestal_events(tmp_path):
+    from ctapipe_io_lst import LSTEventSource
+
+    path = tmp_path / 'pedestal_events.h5'
+    with tables.open_file(path, 'w') as f:
+        data = np.array([(2008, 5), (2008, 11)], dtype=[('obs_id', int), ('event_id', int)])
```

Comment:
Does not really matter for the test, but table created by pedestal finder does not contain obs_id. There is one pedestal_ids file per subrun, so it is not needed.

Reply:
It does contain it since my latest commit....